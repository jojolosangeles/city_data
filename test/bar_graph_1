set -x
rm actual/bar_graph_1.*
python ../code/main.py scripts/bar_graph_1.txt > actual/bar_graph_1.out
diff expected/bar_graph_1.out actual/bar_graph_1.out
diff expected/bar_graph_1.csv actual
diff -b expected/bar_graph_1.png actual
