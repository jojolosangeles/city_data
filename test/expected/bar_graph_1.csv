,Date Occurred,Time Occurred,Area Name,Victim Age,Victim Sex,Victim Descent,Address,Cross Street,Date,Weekday,Year
0,11/04/2010,1345,Central,20.0,F,H,7TH                          ST,LAWRENCE                     ST,2010-11-04,3,2010
1,08/10/2010,1900,77th Street,31.0,F,B,54TH                         ST,3RD                          AV,2010-08-10,1,2010
2,06/09/2010,1925,Pacific,40.0,F,B,MANCHESTER                   BL,BELLANCA                     AV,2010-06-09,2,2010
3,04/06/2010,1445,Topanga,51.0,M,O,ROSITA                       AV,CORBIN                       AV,2010-04-06,1,2010
4,05/23/2010,1805,Southwest,32.0,F,H,43RD                         PL,WALTON                       AV,2010-05-23,6,2010
5,09/02/2010,1520,Foothill,45.0,F,W,9600    SUNLAND                      BL,,2010-09-02,3,2010
6,10/27/2010,1910,77th Street,61.0,F,B,CRENSHAW                     BL,74TH                         ST,2010-10-27,2,2010
7,09/30/2010,540,77th Street,70.0,M,B,NORMANDIE                    AV,MANCHESTER                   AV,2010-09-30,3,2010
8,11/19/2010,1440,77th Street,,M,O,54TH                         ST,NORMANDIE                    AV,2010-11-19,4,2010
9,02/21/2010,300,Newton,44.0,F,O,100 W  52ND                         ST,,2010-02-21,6,2010
10,09/01/2010,1530,Hollywood,,M,H,FERN DELL                    DR,RED OAK                      DR,2010-09-01,2,2010
11,03/11/2010,1840,77th Street,55.0,F,B,VAN NESS                     AV,GAGE                         AV,2010-03-11,3,2010
12,03/23/2010,1400,N Hollywood,45.0,M,W,TUJUNGA                      AV,MOORPARK                     ST,2010-03-23,1,2010
13,06/28/2010,210,Topanga,,M,X,LEDERER,N  VICTORY,2010-06-28,0,2010
14,04/17/2010,1130,Northeast,29.0,F,H,ALDAMA,AVENUE 52,2010-04-17,5,2010
15,03/17/2010,1600,Hollenbeck,,M,H,BOYLE                        AV,5TH                          ST,2010-03-17,2,2010
16,07/25/2010,600,Hollenbeck,41.0,M,O,7TH                          ST,GOLDEN STATE                 FY,2010-07-25,6,2010
17,10/27/2010,1020,West LA,53.0,F,W,OAKHURST                     DR,PICO                         BL,2010-10-27,2,2010
18,02/03/2010,1520,Central,20.0,M,O,4TH                          ST,SPRING                       ST,2010-02-03,2,2010
